libvast/src/system/application.cpp

```diff
@@ -173,6 +174,12 @@ auto make_export_command() {
   return export_;
 }
 
+auto make_get_command() {
+  return std::make_unique<command>(
+    "get", "extracts the events assiciated with ids", "TODO",
+    opts("?get").add<std::string>("format,f", "how to format the output"));
```

Comment:
Can you align the option description and short option with the other format options?